# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../mkcue/install_manpage.patch
# Copyright (C) 2008 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./GNUmakefile.in.orig	2008-01-19 23:29:50.000000000 -0300
+++ ./GNUmakefile.in	2008-01-19 23:31:13.000000000 -0300
@@ -71,6 +71,7 @@
 	rm -rf configure configure.lineno
 
 install: ${DESTDIR}${bindir}/mkcue
+	${INSTALL_DATA} mkcue.1 ${DESTDIR}${mandir}/man1/
 
 ${DESTDIR}${bindir}/%: %
 	${INSTALL_PROGRAM} $< $@
